envoy/network/dns.h

```diff
@@ -56,7 +55,7 @@ class ActiveDnsQuery {
   virtual void addTrace(uint8_t trace) PURE;
 
   /** Return the DNS query traces. */
-  virtual OptRef<const std::vector<Trace>> getTraces() PURE;
+  virtual absl::optional<std::vector<std::string>> getTraces() PURE;
```

Comment:
again why not just return the trace string?  there's no callers who want the individual vectors, and it'd let us use a string instead of an optref vector

Reply:
Sorry, I misread. Done.